src/main/scala/cromwell/backend/BackendValidationActor.scala

```diff
@@ -0,0 +1,50 @@
+package cromwell.backend
+
+import akka.actor.{Actor, ActorLogging}
+import akka.event.LoggingReceive
+import cromwell.backend.BackendValidationActor.{Validate, ValidationResult}
+import wdl4s.NamespaceWithWorkflow
+
+import scala.concurrent.Future
+
+object BackendValidationActor {
+  sealed trait BackendValidationActorMessage
+  case class Validate(namespace: NamespaceWithWorkflow, wfInputsJson: Option[String] = None, wfOptionsJson: String) extends BackendValidationActorMessage
```

Comment:
I don't think it's happening right now in either of the backends, but the general opinion here was that it may be a good-to-have to allow people to validate, say, if an input exists or not.